contracts/gateway/AxelarAmplifierAuth.sol

```diff
@@ -43,7 +46,18 @@ contract AxelarAmplifierAuth is Ownable, BaseWeightedMultisig, IAxelarAmplifierA
     function rotateSigners(bytes calldata newSigners) external onlyOwner {
```

Comment:
Since the auth interface is more generic. Although, I'm thinking of inheriting auth in gateway instead since the abstraction doesn't seem as useful anymore